### ISSUE TITLE
Logging in when offline

### ISSUE THREAD

**Issue by Diusrex** [CMPUT301W15T10/301-Project#83]
When the user is offline, we should not prompt them to enter their username. Instead, there should be a toast saying "Logged in as X" if they previously logged in, or some toast/warning that they must first connect online.

**Comment by blainelewis1** [comment]
I disagree, I think instead if anything we should store all users to offline login

**Comment by Diusrex** [comment]
That is probably a better way to do it. What I definitely think is that it shouldn't ask them to log in, then not even care about what they entered if there is no internet connection

**Comment by blainelewis1** [comment]
It might be possible to save all users locally and then just push the new user to the server when they're online...
